### PR TITLE
Bugzilla 1316593: Adding information to Reclaiming Resources section

### PR DESCRIPTION
**Pull request by ghost**
@openshift/team-documentation PTAL.

https://bugzilla.redhat.com/show_bug.cgi?id=1316593

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
[rev_history]
|xref:../install_config/persistent_storage/persistent_storage_nfs.adoc#install-config-persistent-storage-persistent-storage-nfs[Configuring Persistent Storage -> Using NFS]
|Updated the configuration examples in xref:../install_config/persistent_storage/persistent_storage_nfs.adoc#nfs-reclaiming-resources[Reclaiming Resources] section.
%

File: install_config/persistent_storage/persistent_storage_nfs.adoc [hunk 1]
```diff
@@ -371,12 +370,64 @@ NFS implements the {product-title} *Recyclable* plug-in interface. Automatic
...
 volume.
 
-By default, persistent volumes are set to *Retain*. NFS volumes which are set to
+By default, PVs are set to *Retain*. NFS volumes which are set to
 *Recycle* are scrubbed (i.e., `rm -rf` is run on the volume) after being
 released from their claim (i.e, after the user's `*PersistentVolumeClaim*` bound
 to the volume is deleted). Once recycled, the NFS volume can be bound to a new
 claim.
 
+Once claim to a PV is released (that is, the is PVC deleted), the PV object
```
Comment (qinpingli) [inline install_config/persistent_storage/persistent_storage_nfs.adoc]: s/the is PVC deleted/the PVC is deleted/

File: install_config/persistent_storage/persistent_storage_nfs.adoc [hunk 2]
```diff
@@ -371,12 +370,64 @@ NFS implements the {product-title} *Recyclable* plug-in interface. Automatic
...
+
+Deleting the original PV and re-creating it with the same name is discouraged.
+Attempting to manually change the status of a PV from `Released` to `Available` will
+cause errors and potential data loss.
+
+[NOTE]
+====
+A PV with retention policy of `Recycle` will scrub (`rm -rf`) the data and
+mark it as `Available` for claim. The `Recycle` retention policy is deprecated in
+future versions and should be avoided.
```
[the remark this answers is not in the document]
Comment (qinpingli) [reply install_config/persistent_storage/persistent_storage_nfs.adoc]: About from which version volume recycler is deprecated: https://github.com/openshift/openshift-docs/blob/master/release_notes/ocp_3_6_release_notes.adoc

**Comment by tmorriso-rh** [comment]
@qinpingli Changes made, PTAL.

**Comment by qinpingli** [comment]
LGTM

**Pull request merged**